srcs/ConfigGroup.cpp

```diff
@@ -108,6 +119,35 @@ std::vector<Config> &ConfigGroup::getConfig(int index)
 {
     return (_configs[index]);
 }
+bool ConfigGroup::checkDupLocation(std::vector<Config> server_vector)
+{
+	int size = server_vector.size();
+
+	for (int i = 0; i < size; i++)
+	{
+		for (int j = 0; j < size - i; j++)
+		{
+			if ((j + 1 < size) && (server_vector[j].location_path == server_vector[j + 1].location_path))
+				return false;
+		}
+	}
```

Comment:
```suggestion
	for (int i = 0; i < size - 1; i++)
	{
		for (int j = i; j < size; j++)
		{
			if (server_vector[i].location_path == server_vector[j].location_path)
				return false;
		}
	}
```

요런식으로 수정하셔야 모든 블록들을 비교할 수 있을 것 같아요.

Reply:
재수정해서 커밋했습니다.